vale/styles/vocab.txt

```diff
@@ -24,6 +24,7 @@ Dockerfile
 Enum
 Eventbridge
 Explainshell
+Fastlane
```

Comment:
If this is https://fastlane.tools/, change this to `fastlane` - again, match brand capitalisation.

Even Google's style guide recommends this https://developers.google.com/style/product-names